.storybook/preview.tsx

```diff
@@ -79,15 +79,13 @@ export const parameters: Preview['parameters'] = {
     ],
   },
   badgesConfig: {
-    ...createInitialReleaseConfig('2.0'),
-    ...createInitialReleaseConfig('1.3'),
-    ...createInitialReleaseConfig('1.2'),
-    ...createInitialReleaseConfig('1.1'),
-    ...createInitialReleaseConfig('1.0'),
-    ...createCurrentReleaseConfig('1.0'),
-    ...createCurrentReleaseConfig('1.3'),
-    ...createCurrentReleaseConfig('2.0'),
-    ...createCurrentReleaseConfig('2.1'),
+    ...createComponentVersion('2.0'),
+    ...createComponentVersion('1.3'),
+    ...createComponentVersion('1.2'),
+    ...createComponentVersion('1.1'),
+    ...createComponentVersion('1.0'),
+    ...createThemeVersion('1.0'),
+    ...createThemeVersion('2.0'),
```

Comment:
why did theme version 1.3 and 2.1 disappear?

Reply:
Good question. The previous version markers were mixing two ideas together, so the version numbers were ambiguous. 

Based on how design is defining the theme versions, we only have two 1.0 (SLP), and 2.0 (Render). This is mostly to normalize what versions show up for which components (there was no 1.3 theme version; that "1.3" was referring to the components API, hence the API version numbers still having the minors show up)

Reply:
added the above to the description, for that missing clarity!